pyface/ui/wx/util/image_helpers.py

```diff
@@ -33,7 +33,6 @@ class AspectRatio(IntEnum):
 
 def image_to_bitmap(image):
     """ Convert a wx.Image to a wx.Bitmap.
-
```

Comment:
Don't we normally have a new line in the function docstring between the description and the parameters?

Reply:
Yes.  Not sure what happened here.  It might be a bad merge.